orchestrator/migrations/versions/schema/2025-02-20_68d14db1b8da_make_workflow_description_mandatory.py

```diff
@@ -0,0 +1,41 @@
+"""Make workflow description mandatory.
+
+Revision ID: 68d14db1b8da
+Revises: bac6be6f2b4f
+Create Date: 2025-02-20 16:39:34.889953
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "68d14db1b8da"
+down_revision = "bac6be6f2b4f"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    # backfill description with emoty string, just to be sure
+    conn = op.get_bind()
+    backfill_wf_description_with_empty_string(conn)
```

Comment:
Probably best to just catch the error and log a warning for the users. They need to create a migration to backfill the workflow description before applying this migration.